types/types.proto

```diff
@@ -78,3 +78,14 @@ message SimulationResponse {
   GasInfo gas_info = 1 [(gogoproto.embed) = true, (gogoproto.nullable) = false];
   Result  result   = 2;
 }
+
+// CompactBitArray is an implementation of a space efficient bit array.
+// This is used to ensure that the encoded data takes up a minimal amount of
+// space after proto encoding.
+// This is not thread safe, and is not intended for concurrent usage.
+message CompactBitArray {
```

Comment:
Why is it preferable for this to be in `types/` rather than `crypto/types`?

Reply:
In the last pr i put it in the crypto/types dir and was asked to move it to /types. I do feel it is more appropiate in crypto types as its only used there

Reply:
@alexanderbez are you fine with keeping this in crypto/types as it currently is? I'd rather not pollute types/ with it.

Reply:
I don't have a strong preference :)

Reply:
Let's stick with crypto/types @marbar3778